### ISSUE TITLE
manque unités 500? 200? 200? dans rfrench/4.4.0/src/library/grDevices/po R-fr.po line 202

### ISSUE THREAD

**Issue by ChristianWia** [phgrosjean/rfrench#42]
Dans  rfrench/4.4.0/src/library/grDevices/po R-fr.po    line 202

msgid "Font file longer than 500 will be truncated"
msgstr "Le fichier de police supérieur à 500 sera tronqué"

msgid "Font family longer than 200 will be truncated"
msgstr "Les familles de fontes supérieures à 200 seront tronquées"

msgid "PostScript font name longer than 200 will be truncated"
msgstr "Le nom des polices PostScript supérieur à 200 sera tronqué"

c'est ambigu on ne sait pas ce que l'on compte: un nombre d'octets ?  une liste de polices ? un nombre de caractères ?  

Ca serait mieux:
msgid "Font file longer than 500 bytes will be truncated"
msgid "Font family longer than 200 members will be truncated"
msgid "PostScript font name longer than 200 characters will be truncated"

A faire remonter pour la mise a jour des sources je pense.

**Comment by phgrosjean** [comment]
Oui, ce n'est pas précisé. Mais il me semble que ce n'est pas forcément important. L'essentiel est que le message indique que le fichier est trop volumineux pour R. Ceci dit, en relisant, je préfèrerais "Le fichier de police de taille supérieure à 500 sera tronqué" plutôt que "Le fichier de police supérieur à 500 sera tronqué".
Ici, on traduit, on ne réécrit pas les messages de R... mais si tu penses que c'est important de préciser, on peut garder cela dans la liste des items à faire remonter au R Core Team.
